syntax = "proto3";

option go_package = "github.com/CompressedWater/grpc_py_go_test/proto";

service Greeter {
  rpc SayHello (HelloRequest) returns (HelloReply) {}
  rpc SayHelloAgain (HelloRequest) returns (HelloReply) {}
}

message HelloRequest {
  string name = 1;
  string cur_date = 2;
}

message HelloReply {
  string message = 1;
  string cur_date = 2;
}